x-pack/plugins/profiling/e2e/setup_profiling_resources.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import axios from 'axios';
+
+export async function setupProfilingResources({
+  kibanaUrlWithAuth,
+}: {
+  kibanaUrlWithAuth: string;
+}) {
+  // eslint-disable-next-line no-console
+  console.log('Setting up Universal profiling resources...');
+  await axios.post(
+    `${kibanaUrlWithAuth}/api/profiling/v1/setup/es_resources`,
+    {},
+    { headers: { 'kbn-xsrf': true } }
+  );
```

Comment:
@dgieselaar what do you think of using Profiling set up API here to add the mappings?

Reply:
Makes sense to me. I'm not sure if you run into issues with the archives, but we'll find out along the way.